ios/browser/api/ads/brave_ads.h

```diff
@@ -54,15 +54,12 @@ OBJC_EXPORT
 /// Whether or not the users current region is supported
 + (BOOL)isSupportedRegion;
 
-/// System info
-@property(nonatomic, class) BraveAdsSysInfo* sysInfo;
-/// The build channel info that ads is configured for
-@property(nonatomic, class) BraveAdsBuildChannelInfo* buildChannelInfo;
-
 #pragma mark - Initialization / Shutdown
 
 /// Initializes the ads service
-- (void)initialize:(void (^)(bool success))completion;
+- (void)initialize:(void (^)(bool success))completion
+             sysInfo:(BraveAdsSysInfo*)sysInfo
+    buildChannelInfo:(BraveAdsBuildChannelInfo*)buildChannelInfo;
```

Comment:
The `completion` argument should be the last argument:

I.e. `- (void)initializeWithSysInfo:(BraveAdsSysInfo*)sysInfo buildChannelInfo:(BraveAdsBuildChannelInfo*)buildChannelInfo completion:(void (^)(bool success))completion;`

Reply:
Fixed